pages/en-us/components/footer.mdx

```diff
@@ -0,0 +1,190 @@
+import { Playground, ExampleBlock, Attributes, Layout } from 'lib/components'
+import { Footer, Text } from 'components'
+
+export const meta = {
+  title: 'footer',
+  group: 'Surfaces',
+}
+
+## Footer
+
+<Playground
+  scope={{ Footer }}
+  code={`
+<Footer>
+  <Footer.Group title="Company">
+    <Footer.Link href="#">Home</Footer.Link>
+    <Footer.Link href="#">About</Footer.Link>
+    <Footer.Link href="#">Careers</Footer.Link>
+    <Footer.Link href="#">Partners</Footer.Link>
+    <Footer.Link href="#">Blog</Footer.Link>
+    <Footer.Link href="#">Next.js Conf</Footer.Link>
+  </Footer.Group>
+  <Footer.Group title="Product">
+    <Footer.Link href="#">Pricing</Footer.Link>
+    <Footer.Link href="#">Vercel for GitHub</Footer.Link>
+    <Footer.Link href="#">Vercel for GitLab</Footer.Link>
+    <Footer.Link href="#">Vercel for Bitbucket</Footer.Link>
+    <Footer.Link href="#">Vercel Edge Network</Footer.Link>
+    <Footer.Link href="#">Integrations Marketplace</Footer.Link>
+    <Footer.Link href="#">Command-Line</Footer.Link>
+  </Footer.Group>
+  <Footer.Group title="Education">
+    <Footer.Link href="#">Documentation</Footer.Link>
+    <Footer.Link href="#">Guides</Footer.Link>
+    <Footer.Link href="#">Support</Footer.Link>
+  </Footer.Group>
+  <Footer.Group title="More">
+    <Footer.Link href="#">Open Source Software</Footer.Link>
+    <Footer.Link href="#">Design System</Footer.Link>
+  </Footer.Group>
+</Footer>
+`}
+/>
+
+<Playground
+  title="Light"
+  desc="Footer in Light Background"
+  scope={{ Footer }}
+  code={`
+<Footer light>
```

Comment:
Should I use `reverse` to declare here?
We don't actually allow the user to customize each `color`, the user needs to define these things via `themes`, and for the footer, just consider if it is the opposite color.